docs/PLUGINS.md

```diff
@@ -27,12 +27,23 @@ containerd allows extensions through two methods:
 
 ### V2 Runtimes
 
-The runtime v2 interface allows resolving runtimes to binaries on the system.
-These binaries are used to start the shim process for containerd and allows
-containerd to manage those containers using the runtime shim api returned by
+containerd supports multiple container runtimes. Each container can be
+invoked with a different runtime. For container runtime interface (CRI) integrations the
+runtimes can be configured in the
+containerd configuration file, including setting a default runtime,
+or can be specified explicitly when creating a container via CRI gRPC
+by selecting the runtime handler to be used selected from the runtimes configured for CRI users.
+
+When a client, such as `ctr` or `nerdctl`, requests of containerd to run a container, the client either will specify the specific
+runtime to use, along with options desired, or, if none is specified, containerd will use that hard-coded default.
```

Comment:
```suggestion
When a client such as `ctr` or `nerdctl` creates a container, it can optionally specify a runtime and options to use.  If a runtime is not specified, containerd will use its default runtime.
```

Reply:
Done